README.md

```diff
@@ -180,6 +180,17 @@ buildkite:
       to: target/package.zip
 ```
 
+## Environment Variables
```

Comment:
I'd be inclined to remove this section or reword it. Without reading `toolbox.mk` this section is going to be quite confusing and if you've read `toolbox.mk` you're unlikely to read this. For a person wondering, "What environment variables are available inside the Toolbox container?" They're just going look at the `docker run` command - and I think that makes it pretty obvious.